shared/people/item/index.js

```diff
@@ -53,17 +55,24 @@ const iconContainerStyle = {marginRight: 20, width: isMobile ? 48 : 32}
 
 const childrenContainerStyle = {
   overflow: 'hidden',
-  paddingRight: isMobile ? 100 : 80,
+  paddingRight: isMobile ? globalMargins.large : globalMargins.medium,
   position: 'relative',
   width: 'auto',
 }
 
-const timestampContainerStyle = {
+const timestampContainerStyleMulti = {
   ...globalStyles.flexBoxRow,
   alignItems: 'center',
+  alignSelf: 'flex-start',
   position: 'absolute',
-  right: isMobile ? globalMargins.tiny : globalMargins.small,
-  top: 12,
+  right: globalMargins.small,
+  top: globalMargins.xsmall,
+}
+
+const timestampContainerStyleSingle = {
+  ...timestampContainerStyleMulti,
+  alignSelf: 'center',
+  top: 'inherit',
```

Comment:
`'inherit'` can't be used on mobile, I'd factor out what's common between `single` and `multi` instead of overriding stuff from `multi` in here